README.md

```diff
@@ -68,36 +68,86 @@ The frontend will display the contents of the results plugin for the AutoGov sta
 
 ### Installation into Backstage
 
+#### Update Configuration
+
+1. Add an autogov config section to your `app-config.yaml`
+
+```yaml
```

Comment:
If this is indented with tabs to be sub-level to the above `1.`, the numbering sequence will be automatically continued.